services/product-service/.env.example

```
@@ -1,18 +1,22 @@
+
+REDIS_SOCKET_CONNECT_TIMEOUT=30.0
+REDIS_SOCKET_TIMEOUT=30.0
 PRODUCT_CACHE_TTL=300
 DEFAULT_PAGE_SIZE=20
 MAX_PAGE_SIZE=100
+
 # OpenTelemetry
+OTEL_ENABLED=true
 OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
 OTEL_EXPORTER_OTLP_INSECURE=true
+OTEL_PYTHON_EXCLUDED_URLS="health"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)